lib/query.js

```diff
@@ -313,6 +313,21 @@ export default class Query {
     return this;
   }
 
+  limitResults(num) {
+    this.limit = num;
+    return this;
+  }
+
+  skipResults(num) {
+    this.offset = num;
+    return this;
+  }
+
+  retrievePage(num) {
+    this.page = num;
+    return this;
+  }
```

Comment:
@akiroz I prefer keeping the function name as `limit(num)`, `offset(num)` and `offset(num)`.